ReadMe.md

```diff
@@ -10,15 +10,28 @@ Requires a connected dev board running Marauder FW. [See install instructions fr
    GitHub action workflow, and select the most recent run, scroll down to artifacts.
 3. The FAP is built for the `dev` and `release` channels of both official and unleashed
    firmware. Download the artifact corresponding to your firmware version.
-4. Extract `ESP32_WiFi_Marauder.fap` from the ZIP file to your Flipper
-   Zero SD card, preferably under Apps/GPIO along with the rest of the GPIO apps.
+4. (Optional step to avoid confusion) Go to "Apps/GPIO" on the Flipper SD Card, delete any existing Marauder app, on some firmwares there will be a `ESP32CAM_Marauder.fap` or similar.
+5. Extract `esp32_wifi_marauder.fap` from the ZIP file downloaded in step 3 to your Flipper Zero SD card, preferably under Apps/GPIO along with the rest of the GPIO apps. (If you're using qFlipper to transfer files you need to extract the content of the ZIP file to your computer before you drag it to qFlipper, as qFlipper does not support direct dragging from a ZIP file (at least on Windows).
 
 From a local clone of this repo, you can also build the app yourself using ufbt.
 
 ## In-app ESP32 flasher (WIP)
```

Comment:
Actually might make more sense to add your name and comment under this heading. (Your call :) )